### ISSUE TITLE
queueWorker check to see if job already finished

### ISSUE THREAD

**Issue by lcorrigall** [unchartedsoftware/PLUTO#538]
If RabbitMQ is shut down before the workers are, and a worker is running a job, then the RabbitMQ message cannot be acknowledged and will be put back on the Queue when RabbitMQ starts up again. Then, another worker will pick up the job and process it again.

To protect against this, have the queueWorker check the run record and see if the job completed before starting the validator, and ack the message if it is already done.
